### DISCUSSION TITLE
Major modifications.

### DISCUSSION THREAD

**Discussion by Carrickview** [Stellarium/stellarium#3129] [General]
In the Commentariolus, Copernicus attributed a third motion to the Earth-

_"The third movement is the declination movement. For the axis of daily rotation is not parallel to the axis of the great circle but is inclined to it by such a part of the circumference, which in our time is almost 23 and a half degrees. Thus the centre of the Earth always remains in the plane of the ecliptic, i.e. on the circumference of a great circle, and its poles revolve, drawing small circles on both sides around the centres equidistant from the axis of the great circle. This movement, too, takes place over a period of almost a year and is almost equal to the revolution of the great wheel" Copernicus, Commentariolus.

https://copernicus.torun.pl/zasoby/archiwum/pisma-tablice-i-noty-astronomiczne/komentarzyk-commentariolus

As the Precession of the Equinoxes within the Ptolemaic framework created the roughly 1-degree variation every 72 years, he needed to alter his views by the time he wrote De Revolutionibus by assigning axial precession to account for that Ptolemaic feature. The Precession of the Equinoxes can be accounted for differently as an extension of the system of reckoning that creates the 365/366-day calendar framework.

It is evident that the planet does not complete 1461 rotations in 4 years, covering the entire calendar framework from March 1st to Feb 29th four years later.  The foundations of timekeeping, unknown to Copernicus and Galileo, are based on the heliacal rising of Sirius and that it skips a first annual appearance by one day after the fourth 365-day cycle-

".. on account of the procession of the rising of Sirius by one day in the course of 4 years,.. therefore it shall be that the year of 360 days and the five days added to their end, so one day shall be from this day after every four years added to the five epagomenae before the new year" Canopus Decree 238 BC.

The third motion, which Copernicus attributed to the Earth's surface in his original Commentariolus description, is close to being correct but requires several significant additions. Galileo sought to disprove the third motion by other means-

"From what I see, did not understand very well- was a certain experiment which I exhibited to some gentlemen there at Rome, and perhaps at the very house of Your Excellency, in partial explanation and partial refutation of the "third motion"[14] attributed by Copernicus to the earth. This extra rotation, opposite in direction to all other celestial motions, appeared to many a most improbable thing,and one that upset the whole Copernican system. . . . What I said was designed to remove a difficulty attributed to the Copernican system, and I later added that anyone who would reflect upon the matter more carefully would see that Copernicus had spoken falsely when he attributed his "third motion" to the earth since this would not be a motion at all, but a kind of rest. It is certainly true that to the person holding the bowl, such a ball appears to move with respect to himself and to the bowl and to turn upon its axis. But with respect to the wan (or any other external thing), the ball does not turn at all, and does not change its tilt, and any point upon it will continue to point toward the same distant object" Galileo, The Assayer.

http://web.stanford.edu/~jsabol/certainty/readings/Galileo-Assayer.pdf

There is a satisfactory conclusion that eluded both Galileo and Copernicus, although Copernicus came quite close in his original perspectives.

**Comment by gzotti** [comment]
Please keep using your original thread #3109.
